apps/ml/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  core_api:
+    build: .
+    container_name: "core-api-container"
```

Comment:
container_name: "core-api-container" Not needed

Reply:
container name at all? I renamed it now for ml-api-container.

Reply:
We don't need this at all :D